EvidenceApi.Tests/GlobalSetup.cs

```diff
@@ -13,11 +13,11 @@ public void SetUp()
     {
         try
         {
-            DotEnv.Config(true, Path.GetFullPath("../../../../.env.example"));
+            DotEnv.Config(true, Path.GetFullPath("../../../../.env"));
```

Comment:
Change because for some reason, the integration tests were using the envars in the .env.example file -- I couldn't see why they should.